src/Browsershot.php

```diff
@@ -716,6 +728,21 @@ protected function createCommand(string $url, string $action, array $options = [
 
         $command['options']['args'] = $this->getOptionArgs();
 
+        if (parse_url($url, PHP_URL_SCHEME) === null) {
```

Comment:
Previously all code would work without this check. Is this extra check necessary?

Reply:
Agreed, I think I shouldn't have to add it. ProcessFailedException are more than enough. I'll remove it.